### DISCUSSION TITLE
Auto Update?

### DISCUSSION THREAD

**Discussion by CaptainDoubleYou** [seerge/g-helper#2362] [Ideas]
I would like to suggest adding an auto update feature similar to OBS Studio. If there is a new version available a pop up will show up and asks you if you would like to update. Currently, I have to download, extract, copy and overwrite the older exe and when launched Windows will ask you would like to run the program on a blue screen background (basically unverified app). Maybe the auto update can help with all of this. I appreciate all the help you put into it. I'm an avid user of your app.

**Comment by seerge** [comment]
@CaptainDoubleYou hello, there is already an auto update feature.

Just get an older version and you will be prompted with an update alert.

Just keep in mind it checks only for final releases, not pre-releases. And this is by design

**Reply by CaptainDoubleYou** [reply]
Hi Seerge,

Do you know which older version to choose? Thank you. That will save me a ton of trouble updating every time.

**Reply by seerge** [reply]
@CaptainDoubleYou any older version will prompt for update to latest 0.164 (on the moment of writing of this comment)
